src/main/java/UseCase/Search.java

```diff
@@ -17,8 +17,10 @@ public class Search implements searchInputBoundary {
     private final searchOutputBoundary searchPresenter;
 
        private GetSongAccessInterface songLibrary;
+    // TODO: Deal with singleton
 
     public Search(searchOutputBoundary searchPresenter){
+        // import songLibrary instead of passing it in
```

Comment:
hasn't this already been imported in through GetSongAccessInterface?

Reply:
oops yeah i think i accidentally copied that in when dealing w merge conflicts